test/cadet/jobs/autograder/grading_job_test.exs

```diff
@@ -20,6 +20,12 @@ defmodule Cadet.Autograder.GradingJobTest do
     end
   end
 
+  defp assert_plagiarism_for(assessments_list) do
+    for {assessment} <- assessments_list do
```

Comment:
The assesments_list is put inside a tuple? Also unless it's nested `Enum.each` or `Enum.map`, I'd prefer using the `Enum` methods since `for` list comprehension can fail silently (when pattern matching failed)